packages/composites/token-allocation/src/config.ts

```diff
@@ -1,7 +1,7 @@
 import Amberdata from '@chainlink/amberdata-adapter'
 import CoinApi from '@chainlink/coinapi-adapter'
 import CoinGecko from '@chainlink/coingecko-adapter'
-import CoinMarketCap from '@chainlink/coinmarketcap-adapter'
+import * as CoinMarketCap from '@chainlink/coinmarketcap-adapter'
```

Comment:
Had to update this as the variables are being exported differently from the coinmarketcap adapter.  Open to suggestions on a better way to do this.

Reply:
Coinmarketcap looks the same as the other adapters.

What error was it hitting?

Reply:
I had to change the way variables were exported to get the tests to work.

```
const server = expose(NAME, makeExecute(), undefined, endpointSelector).server

export { NAME, makeConfig, makeExecute, server }
```

Reply:
This looks like the correct way of importing to me, before with the `export =` syntax, TS was treating it as a "default export" which was one object that contained keys referencing the exports. Now with the `export {}` syntax, TS treats it as an ES module which has named exports like `NAME, makeConfig, makeExecute`. This is what results in the two different import styles we have now.